cli/js/os_test.ts

```diff
@@ -116,6 +116,11 @@ test(function osPid(): void {
   assert(Deno.pid > 0);
 });
 
+test(function testGetPriority(): void {
+  const priority = Deno.getPriority();
+  console.log("priority", priority);
```

Comment:
Maybe validate the priority is greater than -20 and less than 19?  I'm not sure the console.log() adds value to the test.  You could also test that `Deno.getPriority() === Deno.getPriority(0)`

Reply:
yes, this will mean 3 test suites :
- rust tests
- cli ts tests
- node ts tests

but if one day we drop node api, we need to have node layer. i'll almost copy node tests to test the same features